tools/serverpod_cli/lib/src/generator/dart/library_generators/library_generator.dart

```diff
@@ -660,7 +661,16 @@ class LibraryGenerator {
                   .index(literalString(endpoint.name))
                   .nullChecked,
               'methodConnectors': literalMap(
-                {..._buildMethodConnectors(endpoint, endpoint.methods)},
+                {
+                  ..._buildMethodConnectors(
+                    endpoint,
+                    endpoint.methods.where((method) => !method.isStream),
+                  ),
+                  ..._buildMethodStreamConnectors(
+                    endpoint,
+                    endpoint.methods.where((method) => method.isStream),
```

Comment:
Discussion: Referencing back to the isStream discussion, as we can see here we define what !isStream means. I think this would add additional context if that variable was an enum rather than a bool.